packages/playground/package.json

```diff
@@ -8,15 +8,14 @@
     "dev:razzle": "razzle start",
     "start": "concurrently -c \"blue,cyan\" \"pnpm:dev:tsc\" -k \"pnpm:dev:razzle\"",
     "start:prod": "NODE_ENV=production node build/server.js",
-    "build": "tsc -b && razzle build",
+    "build": "tsc -b && razzle build --noninteractive",
```

Comment:
maybe is not a good solution

Reply:
I have google it but haven't found better solution so far.

Reply:
removed.